e2e/LongHaul/device/Parameters.cs

```diff
@@ -34,7 +40,7 @@ internal class Parameters
             "Transport",
             Default = TransportType.Mqtt,
             Required = false,
-            HelpText = "The transport to use for the connection.")]
+            HelpText = "The transport to use for the connection (i.e. Mqtt, Amqp).")]
```

Comment:
very nit but perhaps `e.g.` is more gramatically correct

Reply:
actually, if `amqp` and `mqtt` are the only options then `i.e.` works

Reply:
They are the only options.